docs/Functions.md

```diff
@@ -2,10 +2,17 @@
 
 > This documentation is incomplete.
 
-Functions use the following syntax.
+Functions are declared using the `fn` keyword. Its arguments are type annotated, with the type following the name of the formal argument. 
+If the function returns a value, the return type must be specified after the arguments are defined, but before the scope.
+The scope contains the functions code. It begins at `{` and ends at `}`.
+Use the `return` keyword to return from the function, and if you are returning a value, it is followed by the variable name. 
```

Comment:
> it is followed by the variable name. 

This is not correct as shown in your example where you return *"inline data"*, i'm not sure if that's the real name but it already better than the variable.  
I hope you know the right definition of this kind of action :).

Reply:
I ended up just changing the code, because in most functions a variable is returned rather than the result of an operation.

Reply:
Sure we can always change it later on :)